api/local.ts

```diff
@@ -18,39 +37,68 @@ import routes from './routes'
  * Only used Locally
  */
 import * as tunnel from './lib/tunnel'
-import * as log4js from 'log4js'
 
 const FileStore = sessionFileStore(session)
 
 const app = express()
 
+/**
+ * Allows us to integrate the Azure key-vault flex volume, so that we are able to access Node configuration values.
+ */
+const mountedSecrets = propertiesVolume.addTo({})
+
+/**
+ * If there are no configuration properties found we highlight this to the person attempting to initialise
+ * this application.
+ */
+if (!getEnvironment()) {
+  console.log(ERROR_NODE_CONFIG_ENV)
+}
+
+/**
+ * TODO: Implement a logger on the Node layer.
+ */
+console.log(environmentCheckText())
+
+// TODO: Testing that we can get the environment variables on AAT from the .yaml file
+console.log('COOKIE_TOKEN')
+console.log(process.env.NODE_CONFIG_ENV)
+console.log(getConfigValue(COOKIE_TOKEN))
+console.log(getConfigValue(COOKIES_USERID))
+console.log(getConfigValue(MAX_LINES))
+console.log(getConfigValue(SERVICES_CCD_DATA_API_PATH))
+console.log(getConfigValue(SERVICES_CCD_DEF_API_PATH))
+console.log(getConfigValue(SERVICES_IDAM_API_PATH))
+console.log(getConfigValue(SESSION_SECRET))
+console.log(getConfigValue(IDAM_CLIENT))
+console.log(getConfigValue(JURISDICTIONS))
+console.log('s2sSecret', getS2sSecret(mountedSecrets))
+console.log('idamSecret', getIDamSecret(mountedSecrets))
+
 app.use(
   session({
     cookie: {
       httpOnly: true,
       maxAge: 1800000,
-      secure: config.secureCookie !== false,
+      secure: getConfigValue(SECURE_COOKIE) !== false,
     },
     name: 'jui-webapp',
     resave: true,
     saveUninitialized: true,
-    secret: config.sessionSecret,
+    secret: getConfigValue(SESSION_SECRET),
     store: new FileStore({
-      path: process.env.NOW ? '/tmp/sessions' : '.sessions',
+      path: getConfigValue(NOW) ? '/tmp/sessions' : '.sessions',
     }),
   })
 )
 
 /**
  * Used Client side
  */
-if (config.proxy) {
+if (getConfigValue(PROXY_HOST)) {
```

Comment:
you don't need this condition as it's already there in the method

Reply:
Done